src/player-wrapper.js

```diff
@@ -461,6 +467,7 @@ PlayerWrapper.prototype.onAdError = function(adErrorEvent) {
  */
 PlayerWrapper.prototype.onAdBreakStart = function() {
   this.contentSource = this.vjsPlayer.currentSrc();
+  this.contentSourceType = this.player.currentType();
```

Comment:
Sorry for the long delay here - I couldn't figure out why those tests are failing, and I finally found it - this needs to be `this.vjsPlayer.currentType()` instead of  `this.player.currentType()`